### PR TITLE
Add README.md file

### PR DESCRIPTION
**Pull request by ShellyGolden**
Adding information about Xpense project

### CODE REVIEW

**Review by LiorKGOW — CHANGES_REQUESTED** [review]
nicely written @ShellyGolden  
maybe you should add the names of the participants in our group :)

**Comment by ShellyGolden** [comment]
Thanks @LiorKGOW 

I added the list

**Review by LiorKGOW — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Xpense
+
+The Xpense project is an open source platform for housekeeping.
+It’s main objective is to monitor housekeeping related income and expenses.
```
Comment (bregman-arie) [inline README.md:4]: Its

**Comment by yuvalk** [comment]
please squash commits

**Comment by yuvalk** [comment]
commit msg subject (first line) should be phrased as "if applied this commit will ..." (add README.md)
then subject can say something about the WHY 
you dont usually need to describe the content, as that is visible from the commit diff.
in special cases, you can add a second paragraph with such explanations (when you think it's appropriate)
and in complicated cases, I sometimes add a 3rd paragraph with even more technical explanations and references.

remember, you should think of the git history as of a novel, you should be telling the story of the code

**Review by ronyyosef — APPROVED** [review]

**Comment by ShellyGolden** [comment]
I squashed the commits, and changed the commit msg
@yuvalk

**Review by LiorKGOW — APPROVED** [review]
looks better as one commit :)
GJ Shelly

**Comment by shlior7** [comment]
#6 issue to resolve

**Pull request merged**